src/modules/admin/components/formRules/NewFormRuleDialog.tsx

```diff
@@ -0,0 +1,340 @@
+            if (conditionsAvailable.length > 0) {
+              const conditionType = conditionsAvailable[0];
+              const defaultValue = pickListMap[conditionType][0].code;
+              // We display all conditions with a non-null value, so setting this condition's value
+              // to some default value for its type (e.g. the first Project in the list) causes it to appear.
+              setRule({ ...rule, [conditionType]: defaultValue });
+            }
+          }}
+          addItemText='Add Condition'
+          disableAdd={conditionsAvailable.length === 0}
+        >
+          {conditions
+            .filter((condition) => !!condition.value)
+            .map(({ conditionType, value }, index) => {
+              return (
+                <RemovableCard
+                  key={conditionType}
+                  onRemove={() => {
+                    // setting this condition's value to undefined will remove it from the modal
+                    setRule({ ...rule, [conditionType]: undefined });
+                  }}
+                  removeTooltip={'Remove Condition'}
+                  sx={{
+                    backgroundColor: theme.palette.grey[100],
+                    border: 0,
+                    py: 1,
+                  }}
+                >
+                  <Grid container spacing={1}>
+                    <Grid item xs={2} sm={1}>
+                      <FormRuleLabelTypography>
+                        {index === 0 ? 'If' : 'And'}
+                      </FormRuleLabelTypography>
+                    </Grid>
+                    <Grid item xs={9} sm={4}>
+                      <FormSelect
+                        disableClearable={true}
+                        value={{ code: conditionType }}
+                        options={conditionPickList}
+                        onChange={(_event, option) => {
+                          if (isPickListOption(option)) {
+                            const newCondition = option.code as ConditionType;
+                            setRule({
+                              ...rule,
+                              [conditionType]: undefined,
+                              [option.code]: pickListMap[newCondition][0].code,
+                            });
+                          }
+                        }}
+                      />
+                    </Grid>
+                    <Grid item xs={2} sm={1}>
+                      <FormRuleLabelTypography
+                        sx={isTiny ? {} : { textAlign: 'center' }}
+                      >
+                        is
+                      </FormRuleLabelTypography>
+                    </Grid>
+                    <Grid item xs={10} sm={5}>
+                      <FormSelect
+                        // disable clearing the input because we expect users to remove the whole condition
+                        disableClearable={true}
+                        value={value ? { code: value } : undefined}
```

Comment:
- If `value` is allowed to be an empty string (re: other suggestion) the initial value should be set to null, not undefined. undefined indicates that its an uncontrolled component so you get an error.
- Suggest adding placeholder text
```suggestion
                        disableClearable
                        value={value ? { code: value } : null}
                        placeholder={`Select ${startCase(
                          conditionType.replace(/Id$/, '')
                        )}`}
```